src/commands/synthetics/utils.ts

```diff
@@ -643,10 +647,6 @@ export const getTestsToTrigger = async (
     )
   }
 
-  if (waitedTests.length > 0) {
-    reporter.testsWait(waitedTests)
-  }
```

Comment:
With [this condition](https://github.com/DataDog/datadog-ci/blob/cdb7882d763ff248d42e874562a70e0791512944/src/commands/synthetics/run-tests-lib.ts#L108-L110), we'll still not call `reporter.testsWait()` if there are no tests to run.